src/integrations/action-handlers/test-action.js

```diff
@@ -31,7 +31,7 @@ export async function processAction(
   // might want the request library loaded above
   const contact = await r
     .knex("campaign_contact")
-    .where("campaign_contact_id", campaignContactId);
+    .where("campaign_contact.id", campaignContactId);
```

Comment:
actually, since we're changing this, let's change to `cacheableData.campaignContact.load(campaignContactId)`

Reply:
SHA e54abc7e2ea7f163932038e8aad3131c028dab8e